### PR TITLE
Frontend Unit Tests (Cover, ListPreview, LoginForm, Login)

### PR DESCRIPTION
**Pull request by mikesmith**
Added Frontend Component Unit Tests for:
- Cover
- ListPreview
- LoginForm
- Login

Send over any feedback or anything that needs to change. Thanks!

Slowly chipping away at the frontend coverage. Related to #19.

### CODE REVIEW

File: frontend/src/modules/app/components/login/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import Error from "../errors/";
 import { LoginForm } from "./Form";
 import "./styles.css";
 
-class LogIn extends React.Component {
+export class LogIn extends React.Component {
```
Comment (mikesmith) [inline frontend/src/modules/app/components/login/index.js]: Hmm.. after create the PR, I'm a little concerned with this change. Doing this allows unit testing without having to worry about a mock redux store. However, it seems `/landing/index.js` imports `LogIn` as a named export:

``` javascript
import { Backdrop, Footer, Register, LogIn } from '../app/components/';
```
Landing unit tests are working still. I tested it manually with the backend and logging in seems to work still. Doing a quick debug console.log shows Landing still using `Connect(LogIn)` which is what it always has been using. 

Maybe I am misunderstanding the import/export? Any thoughts?

Comment (mikesmith) [reply frontend/src/modules/app/components/login/index.js]: So. I think I figured out why it is working the way it is.

`landing/index.js` and `Landing.test.js` is importing `LogIn` using the following import:
``` javascript
import { Backdrop, Footer, Register, LogIn } from '../app/components/';
```
This is importing from `../app/components/` which I found has an `index.js` that contains the following export (among other exports):
``` javascript
export { default as LogIn } from "./login";
```
So... `landing/index.js` **is** importing a named export, but the named export is actually the default export from `components/login/index.js` which is the connected component.
 

On the other hand, `login/Login.test.js` is using:
``` javascript
import { LogIn } from '../login';
```
which is the actual named export from `/login/index.js` which is the non-redux component.

Hope that makes sense. 🤷‍♂

**Comment by mikesmith** [comment]
I think I figured out one of the concerns I had. Let me know if there is anything I need to change. Thanks!

**Comment by danielgrijalva** [comment]
@mikesmith hey, sorry for the absence. This is good to go so I'll merge now. Thanks! 💯

**Pull request merged**